### PR TITLE
Update draft-ietf-tls-grease-latest.xml

### PR DESCRIPTION
**Pull request by jsalowey**
Adding security considerations for fingerprinting.

### CODE REVIEW

**Comment by jsalowey** [comment]
This change is to address #5

File: draft-ietf-tls-grease-latest.xml [hunk 1]
```diff
@@ -382,6 +382,11 @@ value twice.</t>
      feature disabled. This allows an active attacker to silently disable the
      new feature. By preventing a class of such interoperability problems,
      GREASE reduces the need for this kind of fallback.</t>
+     
+     <t>If an implementation does not select grease values at random it is
```
Comment (davidben) [inline draft-ietf-tls-grease-latest.xml]: Nit: The rest of the doc styles "grease values" as "GREASE values".

**Review by davidben — APPROVED** [review]
Thanks! Sorry I didn't get to this last week.

**Comment by seanturner** [comment]
@jsalowey if you fix this nit up then @davidben can merge and we can WGLC this draft.

File: draft-ietf-tls-grease-latest.xml (hunk 1 above)
Comment (jsalowey) [reply draft-ietf-tls-grease-latest.xml]: OK I think I committed a fix to the pull request with the correct capitalization.

**Review by davidben — APPROVED** [review]

**Pull request merged**